### PR TITLE
Fix #223 Custom 404, 403, 500 error pages

### LINKED ISSUE #223: Create custom 404, 403, 500 error pages
[Django documentation](https://docs.djangoproject.com/en/1.11/topics/http/views/#customizing-error-views)

### PR DESCRIPTION
**Pull request by verma-varsha**
Fixes #223 . Please let me know if there is any further customization/designing required as far as the HTML templates are concerned.

### CODE REVIEW

File: systers_portal/templates/403.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block title %} - Permission Denied{% endblock %}
+
+{% block content %}
+  
+  <div class="jumbotron jumbotron-custom">
+    <h1>403</h1>
+    <h1>Permission Denied</h1>
```
Comment (tapasweni-pathak) [inline systers_portal/templates/403.html]: Add a message for the user, something like "You don't have authorization to access the page". Also add a back button below the message, on onClick just say ```history.go(-1); return false;```. If you know a better way please go ahead.

File: systers_portal/templates/500.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block title %} - Server Error{% endblock %}
+
+{% block content %}
+  
+  <div class="jumbotron jumbotron-custom">
+    <h1>500</h1>
+    <h1>Server Error</h1>
```
Comment (tapasweni-pathak) [inline systers_portal/templates/500.html]: r/Server Error/Internal Server Error plus what I said above.

**Comment by verma-varsha** [comment]
I don't think that we need to overwrite the default callers in views.py because django by default searches for the custom error templates as '404.html', '403.html' and '500.html' and if it finds one, renders them automatically as happened in my case.

File: systers_portal/templates/403.html (hunk 1 above)
Comment (verma-varsha) [reply systers_portal/templates/403.html]: Should there be a back button on each of the error pages?

File: systers_portal/templates/500.html (hunk 2 above)
Comment (verma-varsha) [reply systers_portal/templates/500.html]: "r/Server Error/Internal Server Error" 
Are you talking about the error message here?

File: systers_portal/templates/403.html (hunk 1 above)
Comment (tapasweni-pathak) [reply systers_portal/templates/403.html]: in all pages.

**Comment by tapasweni-pathak** [comment]
> I don't think that we need to overwrite the default callers in views.py because django by default searches for the custom error templates as '404.html', '403.html' and '500.html' and if it finds one, renders them automatically as happened in my case.

Aah, I am not able to, without urlconf/views.py changes. Can you please help me to reproduce the output as it is appearing in your local? Also have you changed debug to false somewhere in your local. 

Please add screenshots in the description of this pull request.

**Comment by verma-varsha** [comment]
Yeah, I have changed DEBUG=False and have added ALLOWED_HOSTS=['*'] in systers_portal/systers_portal/settings/dev.py
Also, I'm attaching the screenshots below.
screenshot from 2017-10-08 18-54-37
screenshot from 2017-10-08 18-56-14
screenshot from 2017-10-08 18-57-48

**Comment by verma-varsha** [comment]
The pages after incorporating the changes suggested in the review look like this (with the back button working as expected).
screenshot from 2017-10-09 01-59-55
screenshot from 2017-10-09 02-00-21
screenshot from 2017-10-09 02-01-02

**Comment by tapasweni-pathak** [comment]
Aah I need to update my self with on/after '16 updates, more and more! Thanks for the work! Can you please also raise an issue that the back button should display the previous URL? Merging this. 🚀

**Pull request merged**